documentation/grafana_config_auth.md

```diff
@@ -18,7 +18,7 @@ Introduced: v4.0.0
 
 | Name                                              | Type          |  Default                    | Description                                                         | Allowed Values
 | ------------------------------------------------ | -------------  | --------------------------- | ------------------------------------------------------------------  | --------------- |
-| `login_cookie_name`                               | String        | `grafana_session`           | The name of the Grafana session cookie.|
+| `login_cookie_name`                               | String        | GrafanaCookbook::CookieHelper.cookie_name  | Session cookie name,  default changed at Grafana 6.2.0 so programatically determining default |
```

Comment:
just realized I didn't update the docs for this.  Had to move this into the install logic, as it was resolving the call at cookbook compile time which means recipes couldn't over-ride the value